### PR TITLE
Fix unborn Ninja

### LINKED ISSUE #158: It's possible to create an unborn children as a ninja

### PR DESCRIPTION
**Pull request by danielsp45**
Closes #158.

### CODE REVIEW

File: lib/bokken/accounts/ninja.ex [hunk 1]
```diff
@@ -46,8 +46,21 @@ defmodule Bokken.Accounts.Ninja do
...
     |> assoc_constraint(:guardian)
     |> assoc_constraint(:user)
     |> unique_constraint(:user_id)
   end
+
+  defp validate_birthday(
+         %Ecto.Changeset{valid?: true, changes: %{birthday: birthday}} = changeset
+       ) do
+    if birthday > Date.utc_today() do
+      add_error(changeset, :birthdate, "can't be in the future")
```
Comment (ruilopesm) [inline lib/bokken/accounts/ninja.ex]:
```suggestion
      add_error(changeset, :birthdate, "Ninja's birthday should be in the past")
```

File: test/bokken/accounts_test.exs [hunk 2]
```diff
@@ -94,4 +96,20 @@ defmodule Bokken.AccountsTest do
       assert %Ecto.Changeset{} = Accounts.change_guardian(guardian)
     end
   end
+
+  describe "Ninjas" do
```
Comment (ruilopesm) [inline test/bokken/accounts_test.exs]:
```suggestion
  describe "ninjas" do
```

Just for consistency 😅

**Review by ruilopesm — CHANGES_REQUESTED** [review]

File: lib/bokken/accounts/ninja.ex [hunk 3]
```diff
@@ -46,8 +46,21 @@ defmodule Bokken.Accounts.Ninja do
...
+    |> validate_birthday()
     |> assoc_constraint(:guardian)
     |> assoc_constraint(:user)
     |> unique_constraint(:user_id)
   end
+
+  defp validate_birthday(
+         %Ecto.Changeset{valid?: true, changes: %{birthday: birthday}} = changeset
+       ) do
+    if birthday > Date.utc_today() do
```
Comment (MarioRodrigues10) [inline lib/bokken/accounts/ninja.ex]: Make sure ninja age is between 7 and 17 years

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

File: test/bokken/accounts_test.exs (hunk 2 above)
Comment (danielsp45) [reply test/bokken/accounts_test.exs]: Sure, on it 💪🏻

File: lib/bokken/accounts/ninja.ex (hunk 3 above)
Comment (danielsp45) [reply lib/bokken/accounts/ninja.ex]: I also thought of that, will implement it 🙏🏻

**Review by MarioRodrigues10 — APPROVED** [review]
LGTM :+1:

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**